src/config/globals.ts

```diff
@@ -18,3 +18,26 @@ export enum YES_NO_VOTES {
 export const WRITE_IN_CANDIDATE_MAX_LENGTH = 40
 export const LOW_BATTERY_THRESHOLD = 0.25
 export const QUIT_KIOSK_IDLE_SECONDS = 5 * 60 // 5 minutes
+export const AMERICA_TIMEZONES = [
```

Comment:
Should this stuff go in `utils/date.ts`?

Reply:
Sounds reasonable. WIll do.